go.mod

```diff
@@ -4,12 +4,15 @@ go 1.12
 
 require (
 	github.com/ethereum/go-ethereum v1.9.9
+	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/gogo/protobuf v1.1.1
 	github.com/howeyc/fsnotify v0.9.0 // indirect
 	github.com/jmoiron/sqlx v0.0.0-20181024163419-82935fac6c1a
 	github.com/magiconair/properties v1.8.0
-	github.com/makerdao/vulcanizedb v0.0.12-rc.1
+	github.com/makerdao/vulcanizedb v0.0.12-rc.1.0.20200212223442-9582deb86491
 	github.com/mattn/go-runewidth v0.0.6
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
I've noticed these two deps seeming to move in and out of the `go.mod` 🤔

Reply:
good call. i may leave them out of this PR the until we can get to the bottom of if this change is necessary/what we want